modules/deploy/pages/deployment-option/cloud/managed-connectors/transforms.adoc

```diff
@@ -0,0 +1,719 @@
+
+| `target.type`
+| The desired timestamp representation: `string`, `unix`, `Date`, `Time`, or `Timestamp`.
+
+| `format`
+| A `SimpleDateFormat`-compatible format for the timestamp. Used to generate the output when `target.type=string` or used to parse the input if the input is a string. Default: `""`.
+
+| `unix.precision`
+| The desired Unix precision for the timestamp: seconds, milliseconds, microseconds, or nanoseconds. Used to generate the output when type=unix or used to parse the input if the input is a Long. Note: This SMT will cause precision loss during conversions from, and to, values with sub-millisecond components. Default: `milliseconds`.
+
+|===
+
+=== Example
+
+Sample configuration:
+
+----
+"transforms": "TimestampConverter",
+"transforms.TimestampConverter.type": "org.apache.kafka.connect.transforms.TimestampConverter$Value",
+"transforms.TimestampConverter.field": "last_login_date",
+"transforms.TimestampConverter.format": "yyyy-MM-dd",
+"transforms.TimestampConverter.target.type": "string"
+----
+
+Before: `1702041416`
+
+After: `2023-12-08`
+
+== TimestampRouter
+
+The TimestampRouter SMT updates the record's topic field as a function of the original topic value and the record timestamp. This is mainly useful for sink connectors, since the topic field is often used to determine the equivalent entity name in the destination system (e.g. database table or search index name).
+
+NOTE: TimestampRouter SMT should be used with sink connectors only.
+
+=== Configuration
+
+|=== 
+| Property key | Description
+
+| `topic.format`
+| Format string which can contain `${topic}` and `${timestamp}` as placeholders for the topic and timestamp, respectively.
+
+| `timestamp.format`
+| Format string for the timestamp that is compatible with `java.text.SimpleDateFormat`.
+|===
+
+=== Example
+
+Sample configuration:
+
+----
+"transforms": "router",
+"transforms.router.type": "org.apache.kafka.connect.transforms.TimestampRouter",
+"transforms.router.topic.format": "${topic}_${timestamp}",
+"transforms.router.timestamp.format": "YYYY-MM-dd"
+----
+
+== ValueToKey
+
+The ValueToKey SMT replaces the record key with a new key formed from a subset of fields in the record value.
+
+=== Configuration
+
+|=== 
+| Property key | Description
+
+| `fields`
+| Comma-separated list of field names on the record value to extract as the record key.
+|===
+
+=== Example
+
+Sample configuration:
+
+----
+"transforms": "valueToKey",
+"transforms.valueToKey.type": "org.apache.kafka.connect.transforms.ValueToKey",
+"transforms.valueToKey.fields": "txn-id"
+----
+
+== SMTs Error handling
+
+By default `Error tolerance` is set to `NONE`, so the connector will fail in case of any exceptions (notably data parsing or data processing errors). To avoid connector crashing in case of data issues, set `Error tolerance` to `ALL` and specify `Dead Letter Queue Topic Name`, as a place where failed messages are redirected.
```

Comment:
```suggestion
By default, `Error tolerance` is set to `NONE`, so the connector fails for any exception (notably, data parsing or data processing errors). To avoid the connector crashing for data issues, set `Error tolerance` to `ALL`, and specify `Dead Letter Queue Topic Name` as a place where failed messages are redirected.
```